README.md

```diff
@@ -18,7 +18,7 @@ We created hummingbot to promote **decentralized market-making**: enabling membe
 | [![binance](https://i.ibb.co/m0YDQLd/Screen-Shot-2019-03-14-at-10-53-42-AM.png)](https://www.binance.com/?ref=10205187) | binance | [Binance](https://www.binance.com/) | * | [API](https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md) | [![Build Status](https://jenkins-hb.coinalpha.com/buildStatus/icon?job=hb_test-exchange_binance&subject=test)](https://jenkins-hb.coinalpha.com/job/hb_test-exchange_binance/) |
 | [![COINBASE](https://i.ibb.co/h9JdGDW/cbp.jpg)](https://pro.coinbase.com/) | coinbase_pro | [Coinbase Pro](https://pro.coinbase.com/) | * | [API](https://docs.pro.coinbase.com/) | [![Build Status](https://jenkins-hb.coinalpha.com/buildStatus/icon?job=hb_test-exchange_coinbase&subject=test)](https://jenkins-hb.coinalpha.com/job/hb_test-exchange_coinbase/) |
 |<img src="https://www.usenix.org/sites/default/files/sponsor_images/huobi_600x240.png" alt="Huobi Global" width="80" height="30" />| huobi | [Huobi Global](https://www.hbg.com) | 1 | [API](https://huobiapi.github.io/docs/spot/v1/en/) | [![Build Status](https://jenkins-hb.coinalpha.com/buildStatus/icon?job=hb_test-exchange_huobi&subject=test)](https://jenkins-hb.coinalpha.com/job/hb_test-exchange_huobi/) |
-|<img src="https://www.trustnodes.com/wp-content/uploads/2019/06/bittrex-1175x500.png" alt="Bittrex International" width="80" height="30" />| bittrex | [Bittrex International](https://international.bittrex.com/) | 1 | [API](https://bittrex.github.io/api/v1-1) |
+|<img src="https://www.trustnodes.com/wp-content/uploads/2019/06/bittrex-1175x500.png" alt="Bittrex Global" width="80" height="30" />| bittrex | [Bittrex Global](https://global.bittrex.com/) | 1 | [API](https://bittrex.github.io/api/v1-1) |
 
```

Comment:
The PR is also now adding changes that were from other pull requests, and not your changes.

Reply:
I revert the particular commit responsible for this